src/main/java/com/team701/buddymatcher/controllers/pairing/PairingController.java

```diff
@@ -3,16 +3,16 @@
 import com.team701.buddymatcher.dtos.pairing.AddBuddyDTO;
 import com.team701.buddymatcher.dtos.pairing.RemoveBuddyDTO;
 import com.team701.buddymatcher.services.pairing.PairingService;
-import io.swagger.annotations.Api;
-import io.swagger.annotations.ApiOperation;
+import io.swagger.v3.oas.annotations.Operation;
+import io.swagger.v3.oas.annotations.tags.Tag;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.http.HttpStatus;
 import org.springframework.http.MediaType;
 import org.springframework.http.ResponseEntity;
 import org.springframework.web.bind.annotation.*;
 
 @RestController
-@Api
+@Tag(name="")
```

Comment:
Nit: Are you able to write the name of the api's here?
so for this one it would be "Pairing"

Reply:
Have added the names to the API's now